src/v/kafka/server/handlers/list_partition_reassignments.cc

```diff
@@ -143,25 +154,22 @@ ss::future<response_ptr> list_partition_reassignments_handler::handle(
 
     resp.data.topics.reserve(request.data.topics->size());
     for (const list_partition_reassignments_topics& tp : *request.data.topics) {
-        ongoing_topic_reassignment topic_reassignment{.name = tp.name};
+        ongoing_topic_reassignment topic_reassignment;
```

Comment:
i'm not sure i understand this change. looking below it seems like this branch:

```
            } else {
                vlog(klog.debug, "No replica set: ntp {}", ntp);
                vlog(klog.debug, "No updates in progress: ntp {}", ntp);
            }
```

could be taken and then `resp.data.topics.push_back(topic_reassignment);` will have an entry in the response without a topic set. is that expected?

Reply:
Good catch!